src/tools/java/mekanism/tools/common/item/ItemMekanismPaxel.java

```diff
@@ -189,4 +191,15 @@ public void addToBuilder(ImmutableMultimap.Builder<Attribute, AttributeModifier>
         builder.put(Attributes.ATTACK_DAMAGE, new AttributeModifier(BASE_ATTACK_DAMAGE_UUID, "Tool modifier", getAttackDamage(), Operation.ADDITION));
         builder.put(Attributes.ATTACK_SPEED, new AttributeModifier(BASE_ATTACK_SPEED_UUID, "Tool modifier", material.getPaxelAtkSpeed(), Operation.ADDITION));
     }
+
+    @Override
+    public boolean isCorrectToolForDrops(BlockState state) {
+        // Some modded items may not check the stack sensitive version
```

Comment:
given it's the same code, only one of these should exist (calling the other if absolutely required)

Reply:
Okay, I'll remove the Forge added method since it delegates to this one.